### PR TITLE
Replacing OSSEC with Wazuh inside decoders

### PR DESCRIPTION
**Pull request by pereyra-m**
|Related issue|
|---|
|#787|

#### Description

This PR removes the **/decoders/0200-ossec_decoders.xml** file and migrates all its content to **/decoders/0005-wazuh_decoders.xml**, replacing all the _ossec_ mentions with _wazuh_.

Also, the _regex_ expressions are extended to consider both terms, so the retro-compatibility is maintained.

#### Note
There are expressions that weren't updated yet:
-  `<program_name>^ossec$|^wazuh$</program_name>` inside the **wazuh-alert** decoder

#### Tests

- [x] The rules UT **(runtest.py)** runs successfully

### CODE REVIEW

File: decoders/0005-wazuh_decoders.xml [hunk 1]
```diff
@@ -6,7 +6,51 @@
...
+  <type>wazuh</type>
+  <prematch offset="after_parent">^Agent started:</prematch>
+  <regex offset="after_prematch">^ '(\S+\S)'</regex>
+  <order>extra_data</order>
+  <fts>name, location, extra_data</fts>
+</decoder>
+
+<decoder name="wazuh-alert1">
+   <parent>wazuh</parent>
+   <prematch>^ossec: Alert Level:|^wazuh: Alert Level:</prematch>
```
Comment (palaciosjeremias) [inline decoders/0005-wazuh_decoders.xml:39]: This log is only created in `read_ossecalert()` and `OS_Alert_SendSyslog()`
I think these two methods are only part of Wazuh Manager, so it shouldn't be necessary to keep both versions of the decoder.

File: decoders/0005-wazuh_decoders.xml [hunk 2]
```diff
@@ -6,7 +6,51 @@
...
+   <plugin_decoder>WazuhAlert_Decoder</plugin_decoder>
+</decoder>
+
+<decoder name="wazuh-alert">
+  <program_name>^ossec$|^wazuh$</program_name>
```
Comment (palaciosjeremias) [inline decoders/0005-wazuh_decoders.xml:44]: Probably, this decoder complements the above one. If `wazuh-alert1`, we should analyze if `wazuh-alert` is also only for managers.

File: decoders/0005-wazuh_decoders.xml (hunk 1 above)
Comment (DProvinciani) [reply decoders/0005-wazuh_decoders.xml:39]: +1

File: tools/rules-testing/tests/ossec.ini [hunk 3]
```diff
@@ -1,33 +1,33 @@
...
+[ossec: active response: delete firewall]
 log 4 pass = Sat May  7 03:27:57 CDT 2011 /var/ossec/active-response/bin/firewall-drop.sh delete - 172.16.0.1 1304756247.60385 31151
 
 rule = 602
 alert = 3
 decoder = ar_log
 
-[ossec-logcollector: ignore informational messages at startup]
-log 1 pass = 2015/01/29 21:09:49 ossec-logcollector(1950): INFO: Analyzing file: '/var/log/httpd/error_log'.
+[wazuh-logcollector: ignore informational messages at startup]
```
Comment (DProvinciani) [inline tools/rules-testing/tests/ossec.ini:28]: Did logcollector already change the name from ossec to wazuh? Just wondering about that...

**Review by DProvinciani — CHANGES_REQUESTED** [review]
Please check the comments...

File: decoders/0005-wazuh_decoders.xml (hunk 1 above)
Comment (pereyra-m) [reply decoders/0005-wazuh_decoders.xml:39]: You are right, those logs aren't present in the agents

File: tools/rules-testing/tests/ossec.ini (hunk 3 above)
Comment (pereyra-m) [reply tools/rules-testing/tests/ossec.ini:28]: Only the log message was changed to **wazuh-logcollector**, and this is only a rule test related to that update.

File: decoders/0005-wazuh_decoders.xml (hunk 2 above)
Comment (pereyra-m) [reply decoders/0005-wazuh_decoders.xml:44]: The **program_name** will remain like this until we can be sure totally sure about this.

**Review by palaciosjeremias — APPROVED** [review]
LGTM

**Review by DProvinciani — APPROVED** [review]
Looks good!

**Pull request merged**